airflow/providers/amazon/aws/hooks/athena.py

```diff
@@ -91,8 +91,11 @@ def run_query(
         }
         if client_request_token:
             params["ClientRequestToken"] = client_request_token
+        self.log.info("Running query_context:%s, query:\n%s", query_context, query)
```

Comment:
logging query should be optional.
We had complains/problems about this before... for example: https://github.com/apache/airflow/issues/11618

Thus in DbAPIHook we have:
https://github.com/apache/airflow/blob/2242ea4ec1e48278d7b65bf2dfda8151c912546d/airflow/providers/common/sql/hooks/sql.py#L112

We should avoid automatically logging things that other users might find disrupting

Reply:
+1